nextstrain-pathogen.yaml

```diff
@@ -1,5 +1,10 @@
-# This is currently an empty file to indicate the top level pathogen repo.
-# The inclusion of this file allows the Nextstrain CLI to run the
-# `nextstrain build` from any directory regardless of runtime.
+# This file's *existence* marks the top level of a Nextstrain pathogen repo,
+# which allows `nextstrain build` to be run from any subdirectory of the repo
+# regardless of runtime.  For more details, see
+# <https://github.com/nextstrain/cli/releases/tag/8.2.0>.
 #
-# See https://github.com/nextstrain/cli/releases/tag/8.2.0 for more details.
+# This file's *contents* is the "registration metadata" for the pathogen repo,
+# used by `nextstrain setup` and `nextstrain run`.
+---
+compatibility:
+  nextstrain run: true
```

Comment:
With the release of [Nextstrain CLI 10.3.0](https://github.com/nextstrain/cli/releases/tag/10.3.0), this should be switched to per-workflow compatibility declarations. See format in [zika](https://github.com/nextstrain/zika/blob/873c54ba175ffb3cf5ffb5c01b8f32694d94e928/nextstrain-pathogen.yaml#L9-L15).